lib/index.js

```diff
@@ -539,7 +539,7 @@ function _getPublicKeyForPermissionCheck(publicKey, options, callback) {
   if(typeof publicKey === 'string') {
     publicKey = {id: publicKey || ''};
   }
-  api.getPublicKey(null, publicKey, function(err, publicKey) {
+  api.getPublicKey(publicKey, null, function(err, publicKey) {
```

Comment:
That causes different data to be returned (omits private key data), which presumably, could also be updated. This will potentially become a moot point once we have HSM/SSM support. I think it should just stay as is for now.